documentation.js

```diff
@@ -0,0 +1,410 @@
+/* eslint-disable max-len */
+const documentation = {
+  root:
+    [
+      {
+        method: 'GET',
+        route: '/',
+        headers: '',
+        body: '',
+        description: 'Returns the basic information for the Seeker Division Database.'
+      },
+      {
+        method: 'GET',
+        route: '/general/guidelines',
+        headers: '',
+        body: '',
+        description: 'Returns a list of Seeker Division guidelines.'
+      },
+      {
+        method: 'GET',
+        route: '/general/goals',
+        headers: '',
+        body: '',
+        description: 'Returns the current list of Seeker Division goals.'
+      },
+      {
+        method: 'GET',
+        route: '/general/goals/{code}',
+        headers: '',
+        body: '',
+        description: 'Returns the Seeker Division goals by the code provided in the route.'
+      },
+      {
+        method: 'POST',
+        route: '/general/goals',
+        headers: 'Content-Type: application/json',
+        body: '{ "name": "Arboretum: Second Expansion", "description": "We have outgrown our existing facilities. The Engineering Division will be working on a joint project to build a new Arboretum offshoot. Engineering staff needs advisors to ensure that this project goes more smoothly than the last attempt.", "code": "E200" }',
+        description: 'Returns the current list of Seeker Division goals.'
+      },
+      {
+        method: 'PUT',
+        route: '/general/goals/{code}',
+        headers: 'Content-Type: application/json',
+        body: '{ "name": "[Updated Goal Name]", "description", "[Updated Description]", "code": "[Updated Code]" }',
+        description: 'Replaces the goal with the code provided in the route with the data provided in the body.'
+      },
+      {
+        method: 'PATCH',
+        route: '/general/goals/{code}',
+        headers: 'Content-Type: text/plain',
```

Comment:
Make sure you test these with special characters like `/` and `<` to ensure they don't get encoded weirdly on you.